source/common/access_log/access_log_impl.cc

```diff
@@ -280,8 +280,8 @@ Grpc::Status::GrpcStatus GrpcStatusFilter::protoToGrpcStatus(
 }
 
 LogTypeFilter::LogTypeFilter(const envoy::config::accesslog::v3::LogTypeFilter& config) {
-  for (AccessLogType type : config.types()) {
-    types_.insert(type);
+  for (auto i : config.types()) {
+    types_.insert(config.types(i));
```

Comment:
hmmm, the types() will return a array of int. And the int value is the enum's value. And then you use the int value (enum's value) as index to get the enum value?

Reply:
yeah fixed this in last iteration, we probably had a race condition :D